src/domains/rtd-app/04_apim_api_pi.tf

```diff
@@ -0,0 +1,50 @@
+#
+# New Payment Gateway PRODUCTS
+#
+module "payment_instruments_api_product" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//api_management_product?ref=v3.0.0"
```

Comment:
Could we try to increase version to last available on repo `terraform-azurerm-v3`?